CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## v1.0.0-rc3 (unreleased)
 
-- No changes yet.
+- `Provide` and `Invoke` will now error out if `dig.In` or `dig.Out` structs
+  contain private fields. Previously these fields were ignored which often led
```

Comment:
Go docs usually refer to these as exported vs not exported. There's no mention of private or unexported on https://golang.org/ref/spec. Rephrase to,

> `Provide` and `Invoke` will now fail if `dig.In` or `dig.Out` structs contain fields that are not exported. Previously [..]